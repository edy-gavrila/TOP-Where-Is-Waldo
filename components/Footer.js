import React from "react";

function Footer({ text, links }) {
  return (
    <div className="flex flex-col sm:flex-row items-center justify-between sm:h-8 px-2 bg-purple-700 text-white">
      <p className="text-small sm:text-base mb-2 sm:mb-0">{text}</p>
      {links}
    </div>
  );
}

export default Footer;
