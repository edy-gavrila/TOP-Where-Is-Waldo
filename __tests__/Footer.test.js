import { render } from "@testing-library/react"
import Footer from "../components/Footer"

describe("'Footer' component tests:", () => {
    test("It renders correctly:", () => {
        const {container} =  render(<Footer/>);
        expect(container).toMatchSnapshot();
    })
})