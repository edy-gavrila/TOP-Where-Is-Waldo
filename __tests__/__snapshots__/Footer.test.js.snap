// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`'Footer' component tests: It renders correctly: 1`] = `
<div>
  <div
    class="flex flex-col sm:flex-row items-center justify-between sm:h-8 px-2 bg-purple-700 text-white"
  >
    <p
      class="text-small sm:text-base mb-2 sm:mb-0"
    />
  </div>
</div>
`;
